package.json

```diff
@@ -15,36 +22,71 @@
     "@types/node": "^16.11.39",
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
+    "@types/react-redux": "^7.1.25",
+    "axios": "^1.3.5",
+    "bootstrap": "^5.2.3",
     "classnames": "^2.3.1",
-    "eslint-plugin-jest": "^27.0.1",
+    "eslint-plugin-jest": "^27.2.1",
     "formik": "^2.2.9",
-    "material-react-table": "^0.40.8",
+    "material-react-table": "^1.11.3",
+    "primeicons": "^6.0.1",
+    "primereact": "^9.4.0",
+    "ramda": "^0.29.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-helmet-async": "^1.3.0",
-    "react-hook-form": "^7.34.2",
+    "react-hook-form": "^7.43.9",
+    "react-icons": "^4.8.0",
     "react-redux": "^8.0.2",
-    "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.1",
+    "react-router-dom": "^6.10.0",
+    "@craco/craco": "5.0.1",
     "redux": "^4.2.0",
     "redux-thunk": "^2.4.1",
     "sass": "^1.52.3",
-    "typescript": "^4.7.3",
+    "typescript": "^4.9.5",
```

Comment:
`typescript-plugin-css-modules` was added in [this commit](https://github.com/ntersol/react-starter/pull/2/commits/4ad4396d1910cb29ed3d9a2cb34aa379d240fc56), but I can't find where it was used.  Looking at its documentation, I didn't care for the idea of transferring typescript ideas to CSS.